src/app/editor/sourceHighlighter.js

```diff
@@ -48,12 +46,12 @@ class SourceHighlighter {
         .highlightcode {
           position:absolute;
           z-index:20;
-          background-color: ${style || styles.editor.backgroundColor_DebuggerMode};
+          background-color: ${style || 'var(--text-info)'};
```

Comment:
Is --text-info a variable?  I think its just --info.

Reply:
done